src/main/java/ee/ituk/api/mentor/MentorProfileService.java

```diff
@@ -1,6 +1,5 @@
 package ee.ituk.api.mentor;
 
-import com.google.common.annotations.VisibleForTesting;
```

Comment:
swagger2 dependencyga jäin sellest ilma, aga õnneks funktsionaalsust ei kaotanud

Reply:
Miks seda vaja oli?

Reply:
funktsionaalselt polnudki, see on järgmisele arendajale teavituseks, miks mingi modifier `public` on